directjngine/src/main/java/com/softwarementors/extjs/djn/router/processor/standard/form/upload/UploadFormPostRequestProcessor.java

```diff
@@ -109,16 +109,15 @@ private static String getFileParametersLogString(Map<String, FileItem> fileField
     return result.toString();
   }
 
-  public static ServletFileUpload createFileUploader() {
+  public static ServletFileUpload createFileUploader(final long maxUploadMemorySize) {
```

Comment:
The *memory* term may not make sense after my changes to prevent directjngine from reading the entire file into memory